content/04.results.md

```diff
@@ -153,16 +156,16 @@ The process of peer review includes a number of steps which take variable amount
 Of the most abundant preprint categories microbiology was the fastest to publish (140 days, (137, 145 days) [95% CI]) and genomics was the slowest (190 days, (185, 195 days) [95% CI]) (Figure {@fig:publication_delay_panels}A).
 We did observe category-specific differences; however, these differences were generally modest, suggesting that the peer review process did not differ dramatically between preprint categories. 
 One exception was the Scientific Communication and Education category, which took substantially longer to be peer reviewed and published (373 days, (373, 398 days) [95% CI]).
-This hints that there is a key step or steps within the peer review process that greatly halts preprints in this category.
+This hints that there may be differences in the publication or peer review process or culture that apply to preprints in this category.
 
 Examining peer review's affect on individual preprints, we found a positive correlation between preprints with multiple versions and the time elapsed until publication (Figure {@fig:publication_delay_panels}B). 
 Each new version adds additional 51 days before a preprint is published.
 This time duration seems broadly compatible with the amount of time it would take to receive reviews and revise a manuscript, suggesting that many authors may be updating their preprints in response to peer reviews or other external feedback.
 Furthermore, we found that preprints with large embedding space distances from their corresponding peer reviewed publication took longer to publish (Figure {@fig:publication_delay_panels}C).

```

Comment:
```suggestion
The embedding space allows us to compare preprint and published documents to determine if the level of change that documents undergo relates to the time that it takes them to be published.
Distances in this space are arbitrary and must be compared to reference distances.
We found that that average distance of two randomly selected papers from the bioinformatics category was 5.068, while the average distance of two randomly selected papers from bioRxiv was 6.210.
Preprints with large embedding space distances from their corresponding peer reviewed publication took longer to publish (Figure {@fig:publication_delay_panels}C): each additional unit of distance corresponded to roughly sixteen additional days.

```

Reply:
I moved the order of these sentences around to try to improve flow here. See what you think.